azure-pipelines.yml

```diff
@@ -0,0 +1,164 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+variables:
+  # The $(FlutterToolPath) variable is generated by the FlutterInstall task 
+  # See code in the Flutter Build task https://github.com/aloisdeniel/vsts-flutter-tasks/blob/master/tasks/build/index.ts
+  flutterExecPath: $(FlutterToolPath)/flutter.bat
+
+trigger:
+  batch: 'true'
+  branches:
+    include:
+      - develop
+      - release
+      - master
+  paths:
+    include:
+      - device_calendar/*
+
+stages:
+  # ----------- CI -----------
+  - stage: Build
+    jobs: 
+     - job: BuildAndroidAndIos
+       pool:
+        vmImage: 'macOS-latest'
+       steps:
+        - task: FlutterInstall@0
+          displayName: 'Flutter install'
+          inputs:
+            channel: 'stable'
+            version: 'latest'
+
+        - task: FlutterBuild@0
+          displayName: 'Flutter build - Android'
+          inputs:
+            target: 'apk'
```

Comment:
I wonder if it would be better if we try building aab (`appbundle`) instead. Don't think it matters from the build verifications perspective but I guess aab is the new "way to go" so maybe we should try building it?

Reply:
Will give it a go!